pwchem/protocols/__init__.py

```diff
@@ -39,6 +39,7 @@
 from pwchem.protocols.VirtualDrugScreening.protocol_shape_filtering import ProtocolShapeDistancesFiltering
 from pwchem.protocols.VirtualDrugScreening.protocol_ligand_fingerprints import ProtocolFingerprintFiltering
 from pwchem.protocols.VirtualDrugScreening.protocol_ligand_filter import ProtocolGeneralLigandFiltering
+from pwchem.protocols.VirtualDrugScreening.protocol_ligand_characterization import ProtocolLigandCharacterization
```

Comment:
The protocol should have a related test where its functioning is tested. You can see other example sin the pwchem/tests directory